### DISCUSSION TITLE
The problem with grand temples

### DISCUSSION THREAD

**Discussion by dmyersturnbull** [Keriew/augustus#579] [Ideas]
Grand Templates take so long to build that, when they're completed, their bonuses might only last the last 5% of the game.

I found that building them, while fun, rarely aligns with the win condition. I think they'd be dramatically more worthwhile if they were half the resource cost, perhaps with a ~30% drop in bonuses.

The Grand Template to Neptune is the worst offender. By the time it's finished, the city is perhaps 90% done, so the +2 range is unlikely to be useful.

**Comment by crudelios** [comment]
We are somewhat aware of that and we have plans to make monument building more useful in the future. One step at a time though!

Some map makers are however original and make use of features that require monuments, like having a map without water, requiring you to build a monument that serves as a reservoir in order to have water (the Neptune one, I think)

**Reply by dmyersturnbull** [reply]
Neat! I figured adjusting resource costs and bonus %s would be a pretty simple change. Looking forward to seeing what you have in store.
